decidim-elections/README.md

```diff
@@ -0,0 +1,24 @@
+# Decidim::Elections
+
+The Decidim::Elections is a component that allows users to setup non-cryptographic elections, similarly as the old consultation module. Elections are basically polls/surveys with census access management built-in. This allows registered or non-registered users to directly participate in them.
```

Comment:
> similarly as the old consultation module

I don't know if it is too relevant for newcomers to mention a module that is dead for like 3 or 4 years. @carolromero what do you think?